test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/testproviders/PersistenceTestProvider.java

```diff
@@ -85,6 +90,134 @@ public void testControllerPersistence(WorkbenchKieServerScenario deploymentScena
         }
     }
 
+
+    // Verifies https://issues.redhat.com/browse/RHPAM-2762
+    public void testAdminUserPasswordChange(WorkbenchKieServerScenario deploymentScenario) {
+        final String NEW_PASSWORD = "newpassword1234!";
```

Comment:
This breaks the naming convention. Either move it to static field in the class or renamed it to "newPassword".